src/components/Summary/AlterationSummary.vue

```diff
@@ -85,20 +85,60 @@
         </v-layout>
       </div>
     </template>
+
+    <!-- TODO: Name Translation -->
+
+    <!-- TODO: Pre-existing Company Provisions -->
+
+    <!-- TODO: Share Structure -->
+
+    <!-- Alteration Date and Time -->
+    <div class="ma-6 pb-6">
+      <v-container class="alteration-date-time" :class="{ 'invalid': alterationDateTimeInvalid }">
+        <v-row no-gutters>
+          <v-col cols="3" class="pr-4">
+            <label><strong>Alteration Date and Time</strong></label>
+          </v-col>
+
+          <v-col cols="9" class="pl-4">
+            <p class="info-text">Select the date and time of alteration of your business. You may select a date and
+              time up to 10 days in the future (note: there is an <strong>additional fee of $100.00</strong> to enter
+              an alteration date and time in the future). Unless a business has special requirements, most businesses
+              select an immediate Alteration Date and Time.
+            </p>
+
+            <effective-date-time
+              :currentJsDate="getCurrentJsDate"
+              :effectiveDateTime="getEffectiveDateTime"
+              @dateTimeString="setEffectiveDateTimeString($event)"
+              @isFutureEffective="setIsFutureEffective($event)"
+              @valid="setEffectiveDateValid($event)"
+            />
+
+            <v-card flat class="px-16 pb-8 mt-n12" v-if="isFutureEffective && isEffectiveDateTimeValid">
```

Comment:
Thats a pretty big negative margin, although i'm sure you have a good reason. 
A wise man once told me, anything beyond a few negatives might be some other issue ;)

Reply:
I do :)

I want the effective date component to have proper gutter all around when it's the only thing displayed.

But when THIS block is displayed, I want to pull it up into the effective date component to make them look like a single block.

I didn't think this text belonged in the effective date component (not the intro blurb), which is why it's here in its parent component.

I suppose I could have passed the intro/outro blurbs in as props but I think it's OK for now. We'll see how else we use this effective date component in the future and then maybe change .it.